src/stateless/Chip.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div :class="[theme, size, { active: isActive }] | prefix('chip--')" class="chip" tabindex="0" @click="$emit('click')">
+        <div class="chip__label">{{ label }}</div>
+        <div v-if="metadata" class="chip__metadata">{{ metadata }}</div>
+        <span v-if="isRemovable" @click.stop="$emit('remove')">
+            <icon style="width: 8px; height: 8px" class="chip__remove-btn" name="x-bold"></icon>
+        </span>
+    </div>
+</template>
+
+<script>
+import Icon from './icon.vue'
+
+export default {
+    components: {
+        Icon,
+    },
+    props: {
+        theme: { type: String, default: 'dark' }, // dark | light
+        size: { type: String, default: 'normal' }, // condensed | normal
+        label: { type: String, default: '' },
```

Comment:
Shouldn't this be required?

Reply:
According to `ChipWithMultiselect.vue`, yeah :)